content/IA-update_connections/Overview.textile

```diff
@@ -0,0 +1,293 @@
+---
+title: Connections
+meta_description: "Establish and maintain a persistent connection to Ably using the Realtime Client Library SDK."
+section: realtime
+index: 20
+languages:
+  - javascript
+  - nodejs
+  - ruby
+  - java
+  - swift
+  - objc
+  - csharp
+  - flutter
+jump_to:
+  Help with:
+    - Introduction#intro
+    - Getting started#title
+    - Connection state explained
+    - Handling failures
+    - Connection state recovery
+    - Heartbeats#heartbeats
+    - API Reference#api-reference
+
+
+what are connections, mention states exist
```

Comment:
I think this text was left here by mistake? It actually breaks the docs build, as this is considered to be invalid metadata.